hazelcast/src/main/java/com/hazelcast/internal/management/ManagementCenterService.java

```diff
@@ -125,21 +122,16 @@ public ManagementCenterService(HazelcastInstanceImpl instance) {
         this.timedMemberStateFactory = new TimedMemberStateFactory(instance);
         this.identifier = newManagementCenterIdentifier();
 
-        registerListeners();
+        if (this.managementCenterConfig.isEnabled()) {
+            this.instance.getCluster().addMembershipListener(new ManagementCenterService.MemberListenerImpl());
+            start();
```

Comment:
Actually start() method triggers all thread that are getting statistics and member state. And [TimedMemberStateFactory](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/internal/management/TimedMemberStateFactory.java#L105) checks member state.

I deleted lifecycle listener and moved start() method into constructor because when we trigger partial or force start via rest call to man center, node can not start man center due to this listener. When partial start is proceeding, node state is STARTING not STARTED.